workloads/disk.go

```diff
@@ -45,6 +23,6 @@ func (d *Disk) Convert() gridtypes.Workload {
 			Size: gridtypes.Unit(d.Size) * gridtypes.Gigabyte,
 		}),
 	}
-
-	return workload
+	err := manager.SetWorkload(d.NodeId, workload)
```

Comment:
i believe errors should be checked. maybe for now if there is an error, log some message to indicate that SetWorkload failed and return it.